### ISSUE TITLE
Failure on schema fetching if `simplejson` is installed

### ISSUE THREAD

**Issue by bovee** [onecodex/onecodex#66]
```
  File ".../site-packages/onecodex/api.py", line 169, in _fetch_schema
    client=self)
  File ".../site-packages/requests/models.py", line 884, in json
    self.content.decode(encoding), **kwargs
  File ".../site-packages/simplejson/__init__.py", line 533, in loads
    return cls(encoding=encoding, **kw).decode(s)
  File ".../site-packages/potion_client/converter.py", line 122, in __init__
    JSONDecoder.__init__(self, *args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'encoding'
```

Basically requests uses simplejson if installed which passes an `encoding` parameter through the Flask-Potion JSON decoding class to the `json` library (which doesn't support it). Possible fixes are:
 - submit a patch to use simplejson in Flask-Potion (if installed) 
 - monkey-patch the PotionJSONSchemaDecoder to drop the encoding argument if passed before initializing the `json` JSONDecoder

This may be an upstream flask-potion issue, but the way we're setting this up to handle our schema with all of our monkey-patching may not be a use-case they support? Some more investigation of why the other parts of `flask-potion` don't blow up when `simplejson` is installed is probably warranted.

**Comment by rmzelle** [comment]
Thanks for troubleshooting this for me! I just checked and simplejson is a standard package included with WinPython 3.6.2.0: https://github.com/winpython/winpython/blob/master/changelogs/WinPythonQt5-64bit-3.6.2.0.md

**Comment by polyatail** [comment]
Interesting... so [biosustain/potion](https://github.com/biosustain/potion) pulls its `json` from [flask](https://github.com/pallets/flask), which pulls its `json` from [itsdangerous](https://github.com/pallets/itsdangerous), which chooses `simplejson` if its available.

In contrast, [biosustain/potion-client](https://github.com/biosustain/potion-client) just imports the built-in `json`. In Python 3, this implementation no longer supports the encoding keyword argument.

I'm gonna see if I can get a fix pulled on potion-client which brings it in line with potion by pulling `json` from flask.
